sklearn/utils/tests/test_array_api.py

```diff
@@ -581,10 +581,14 @@ def test_count_nonzero(
 @pytest.mark.parametrize("wrap", [True, False])
 def test_fill_or_add_to_diagonal(array_namespace, device_, dtype_name, wrap):
```

Comment:
Let's put "array_api" in the test name to make it easy to run all array_api related tests using `pytest -k array_api` (as we do on the CUDA CI).
```suggestion
def test_fill_or_add_to_diagonal_with_array_api(array_namespace, device_, dtype_name, wrap):
```

Reply:
I think since we are in `test_array_api.py` all tests here are run with -k 'array_api' - other tests in this file also do not include 'array_api' in the test name.